src/helpers/data.ts

```diff
@@ -76,7 +86,7 @@ export function filterData<TTableRowType extends TableRowType>(
       const header = headers[key];
 
       if (header.isFilterable) {
-        let columnValue = element[header.prop];
+        let columnValue = extractValueFromObject(header.prop,element);
```

Comment:
(similar like the first comment, let's prettify this file so there's a space between `,` and `element`.